### PR TITLE
Update about page to include external sources information

### LINKED ISSUE #1989: Update the About page to add the text from "External sources" page
## Problem

To better organize the information about Openverse, we need to remove the top-level "External sources" navigation link and move all the information from it to the About page.

## Description

Move the information from the External sources page to an appropriate place in the About page. This updated text should only be shown when the `new_header` is on.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by anton202**
## Fixes

Fixes #1989  by @obulat 

## Description

This PR adds the external sources page text to about page.
the extrenal sources text in the about page is displayed only if the new header is available.

## Testing Instructions

1. go to the /about page. make sure there is no exteranl source text displayed.
2. go to /preferences. enable the new header. got to /about page. make sure that the external sources text is displayed

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by anton202** [comment]
thx. i`d like to try it myself. i`ll let you know if i have any questions :)

**Review by obulat — APPROVED** [review]
The snapshots are awesome, thank you, @!

File: src/locales/scripts/en.json5 [hunk 1]
```diff
@@ -85,8 +85,7 @@
...
     },
     planning: {
-      content: 'Currently {openverse} only searches images, with search for audio and video provided through {meta}. But we plan to add additional media types such as open texts and audio, with the ultimate goal of providing access to all 1.4 billion CC licensed and public domain works on the web. All of our code is open source ({frontend}, {api}, {catalog}) and we {community}. You can see what {working}.',
-      meta: 'External Sources',
+      content: 'Currently {openverse} only searches images, with search for audio and video provided through External Sources. But we plan to add additional media types such as open texts and audio, with the ultimate goal of providing access to all 1.4 billion CC licensed and public domain works on the web. All of our code is open source ({frontend}, {api}, {catalog}) and we {community}. You can see what {working}.',
```
Comment (sarayourfriend) [inline src/locales/scripts/en.json5:88]: This paragraph is no longer accurate, it turns out :grin: @obulat Do you think we should update it here or create an issue to resolve it? I think a small change like the following could work okay:

```suggestion
      content: 'Currently {openverse} only searches images and audio, with search for video provided through External Sources. But we plan to add additional media types such as open texts and 3D models, with the ultimate goal of providing access to all 1.4 billion CC licensed and public domain works on the web. All of our code is open source ({frontend}, {api}, {catalog}) and we {community}. You can see what {working}.',
```

**Review by sarayourfriend — APPROVED** [review]
LGTM! I noticed one discrepancy in the text but we could create an issue to fix that instead of doing it here. I'll leave that up to Olga.

**Pull request merged**